vnext/src/Libraries/EventEmitter/NativeEventEmitter.windows.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Can you mark the bit you changed that's windows-specific?
We typically add two lines of comments around the modified block of code:
// [Windows
    forked_code_line
// ]Windows

Reply:
It's less important if we know that it's getting removed soon, since there won't be many PRs needing to resolve conflicts here, but still a good practice.